src/views/Modal.jsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import { useNavigate } from 'react-router-dom';
+import { useTheme } from '@mui/material/styles';
+
+const style = (theme) => ({
+	position: 'absolute',
+	top: '50%',
+	left: '50%',
+	transform: 'translate(-50%, -50%)',
+	width: 400,
+	bgcolor: theme.palette.mode === 'dark' ? '#424242' : 'background.paper',
+	color: theme.palette.text.primary,
+	boxShadow: 24,
+	p: 4,
+	fontSize: 20,
+});
+
+export default function BasicModal({ dataEmpty }) {
```

Comment:
<!-- Suggestions propose improvements to the current subject.  -->
**suggestion ( non-blocking ):** Adding this modal in is very nice! Something to consider for the future is to make it more extensible. For example, you could pass in props like `header`, `body`, and `buttonText` and then use this throughout your codebase.